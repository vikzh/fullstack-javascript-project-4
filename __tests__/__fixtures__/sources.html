<!DOCTYPE html>
<html>
<head>
    <link rel="stylesheet" media="all" href="/assets/application.css">
    <link rel="stylesheet" media="all" href="https://test.com/assets/menu.css">
</head>
<body>

<h1>My test Heading</h1>
<p>My test paragraph.</p>


</body>
</html>
