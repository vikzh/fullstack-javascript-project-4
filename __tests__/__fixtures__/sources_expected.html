<!DOCTYPE html>
<html>
<head>
    <link rel="stylesheet" media="all" href="./_files/application.css">
    <link rel="stylesheet" media="all" href="./_files/menu.css">
</head>
<body>

<h1>My test Heading</h1>
<p>My test paragraph.</p>


</body>
</html>
